config.py

```
@@ -0,0 +1,1246 @@
+
+    @model_validator(mode="after")
+    def check_cors_configuration(self) -> Self:
+        """Check CORS configuration."""
+        # credentials are not allowed with wildcard origins per CORS/Fetch spec.
+        # see https://fastapi.tiangolo.com/tutorial/cors/
+        if self.allow_credentials and "*" in self.allow_origins:
+            raise ValueError(
+                "Invalid CORS configuration: allow_credentials can not be set to true when "
+                "allow origins contains the '*' wildcard."
+                "Use explicit origins or disable credentials."
+            )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)